Lesson 4/kolya.stelmax/script2.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+if [[ "$EUID" -ne 0 ]];
+ then
+  echo "Please run script as root"
+  exit 1;
+fi
+
+if
+
+ dpkg --list | grep nginx;
+
+then
+nginx -v;
+else
+echo "nxing is not installed";
+apt remove nginx;
+
+fi
+
+#task2
+apt install curl gnupg2 ca-certificates lsb-release;
```

Comment:
Reading state information... Done
Package gnupg2 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Unable to locate package curl
E: Package 'gnupg2' has no installation candidate
E: Unable to locate package ca-certificates
E: Unable to locate package lsb-release